src/databricks/labs/ucx/hive_metastore/table_migrate.py

```
@@ -182,3 +184,144 @@ def print_revert_report(self, *, delete_managed: bool) -> bool | None:
+            return True
+
+        except NotFound as err:
+            if "[TABLE_OR_VIEW_NOT_FOUND]" in str(err):
+                logger.error(f"Could not find table {from_table_name}. Table not found.")
+            else:
+                logger.error(err)
+        except PermissionDenied:
+            logger.error(f"error applying permissions for table {to_table_name}")
+        return False
+
+    def _move_view(
+        self,
+        from_catalog: str,
+        from_schema: str,
+        from_table: str,
+        to_catalog: str,
+        to_schema: str,
+        del_view: bool,  # noqa: FBT001
+        view_text: str | None = None,
+    ) -> bool:
+        from_table_name = f"{from_catalog}.{from_schema}.{from_table}"
+        to_table_name = f"{to_catalog}.{to_schema}.{from_table}"
+        try:
+            create_sql = f"CREATE VIEW {to_table_name} AS {view_text}"
+            logger.debug(f"Creating view {to_table_name}.")
+            self._backend.execute(create_sql)
+            grants = self._ws.grants.get(SecurableType.TABLE, from_table_name)
+            if grants.privilege_assignments is None:
+                return True
+            grants_changes = [
+                PermissionsChange(pair.privileges, pair.principal) for pair in grants.privilege_assignments
+            ]
+            self._ws.grants.update(SecurableType.TABLE, to_table_name, changes=grants_changes)
+            if del_view:
+                logger.info(f"dropping source view {from_table_name}")
+                drop_sql = f"DROP VIEW {from_table_name}"
+                self._backend.execute(drop_sql)
+            return True
+        except NotFound as err:
+            if "[TABLE_OR_VIEW_NOT_FOUND]" in str(err):
+                logger.error(f"Could not find view {from_table_name}. View not found.")
+            else:
+                logger.error(err)
+        except PermissionDenied:
+            logger.error(f"error applying permissions for view {to_table_name}")
```

```suggestion
```
don't swallow permission errors